expression/builtin_math_test.go

```diff
@@ -557,7 +557,7 @@ func (s *testEvaluatorSuite) TestConv(c *C) {
 		tp := f.GetType()
 		c.Assert(tp.Tp, Equals, mysql.TypeVarString)
 		c.Assert(tp.Charset, Equals, charset.CharsetUTF8)
-		c.Assert(tp.Collate, Equals, charset.CharsetUTF8)
+		c.Assert(tp.Collate, Equals, charset.CollationUTF8)
```

Comment:
This is a bugfix from this PR, although I am not sure what part of this changset fixed it.

Reply:
My pr also fixed it... :)